pkg/apis/servicecatalog/validation/binding.go

```diff
@@ -23,13 +23,17 @@ import (
 	sc "github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog"
 )
 
+// validateBindingName is the validation function for Binding names.
+var validateBindingName = apivalidation.NameIsDNSSubdomain
```

Comment:
This is all me, following the pattern from upstream.  I can change them to just be the `NameIsDNSSubdomain` - no reason I'm aware of for them to be vars.